specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-10-20/myworkbooks_API.json

```diff
@@ -541,19 +574,11 @@
     "SourceIdParameter": {
       "name": "sourceId",
       "in": "query",
-      "required": true,
+      "required": false,
       "type": "string",
       "description": "Azure Resource Id that will fetch all linked workbooks.",
       "x-ms-parameter-location": "method"
     },
-    "ResourceIdParameter": {
```

Comment:
Why are you removing the definition here? Better keep it to avoid breaking change error, if not necessary,

Reply:
It doesn't exist and it never has.

Reply:
I see. Then actually it's not a breaking change. You may ask for breaking change approval for this case.